pypika/tests/test_selects.py

```diff
@@ -208,25 +208,25 @@ def test_where_field_starts_with(self):
 
         self.assertEqual("SELECT * FROM \"abc\" WHERE \"foo\" LIKE 'ab%'", str(q))
 
-    def test_where_field_contains(self):
-        q = Query.from_(self.t).select(self.t.star).where(self.t.foo.like('%fg%'))
+    def test_where_field_does_not_start_with(self):
```

Comment:
Wait, so you replaced all the like tests with just not like ones? Shouldn't we keep both?

Reply:
Thanks for catching. That was not intentional.